librz/core/project.c

```diff
@@ -24,8 +24,9 @@ RZ_API RZ_NONNULL const char *rz_project_err_message(RzProjectErr err) {
 		return "invalid content encountered";
 	case RZ_PROJECT_ERR_MIGRATION_FAILED:
 		return "migration failed";
-	case RZ_PROJECT_ERR_UNKNOWN:
-		break;
+	case RZ_PROJECT_ERR_ZIP:
```

Comment:
Since we do not use ZIP, maybe make it more generic? `RZ_PROJECT_ERR_COMPRESSION_FAILED`? Also it's better to separate compression from decompression errors.